src/modules/shared/components/Blade/Blade.module.scss

```diff
@@ -33,7 +33,7 @@
 			text-align: center;
 		}
 		@include respond-at(md) {
-			font-size: $font-size-3xl;
+			font-size: $font-size-2xl;
```

Comment:
Blijft line-height hieronder een niet-decimaal getal?

Reply:
Het probleem dat hiermee opgelost moet worden is dat op sommige mac-schermen en een klein beetje gescrolled moet worden binnen de blade. 
![image](https://user-images.githubusercontent.com/85545258/155336877-8db33cfd-7d78-4ece-9bc4-b27d9c3acc9f.png)
Er is dan een opmerking ([deze](https://meemoo.atlassian.net/browse/ARC-194?focusedCommentId=28569)) gekomen dat op een bepaalde break-point de tekst kleiner wordt en dat dan niet meer zo is. Ze willen dat ook voor desktop. Even met Christophe besproken en aangezien dit de langste blade titel is, zou die overal altijd op één lijn moeten staan en dus minimale plaats innemen. Lijkt me geen goed idee om voor één blade de titelgrootte aan te passen, dan is de continuïteit verstoort.
![image](https://user-images.githubusercontent.com/85545258/155337124-7ea38bc6-2e44-48e4-9e68-0d1c8609b3f6.png)

Reply:
Line height is nu `5.6rem`. Wat zou die idealiter moeten zijn?